common/containers/Home/index.jsx

```diff
@@ -36,8 +36,10 @@ export class Home extends Component {
     this.constructor.fetchData(this.props.dispatch, this.props)
   }
 
-  static fetchData(dispatch) {
-    dispatch(updateJWT())
+  static fetchData(dispatch, props) {
+    if (props.lgJWT) {
+      dispatch(updateJWT(props.lgJWT))
+    }
```

Comment:
As mentioned in review for https://github.com/LearnersGuild/idm/pull/209, please just remove `fetchData` in this container. Unnecessary.